consumer_group.go

```diff
@@ -12,34 +12,37 @@ import (
 )
 
 type KafkaConsumerGroup struct {
-	GroupID    string
-	Partitions string
-	BrokerList string
-	Offset     string
-	Verbose    bool
-	BufferSize int
-	Messages   chan *sarama.ConsumerMessage
-	Closing    chan struct{}
-	WaitGroup  sync.WaitGroup
+	GroupID     string
+	Partitions  string
+	BrokerList  string
+	Offset      string
+	Verbose     bool
+	BufferSize  int
+	Messages    chan *sarama.ConsumerMessage
+	Closing     chan struct{}
+	WaitGroup   sync.WaitGroup
+	Version     sarama.KafkaVersion
+	DoneInitial bool
```

Comment:
How about the name: `ReadyState`? `DoneInitial` is not a good name in this case in my opinion